sdk-ingestion/src/main/java/care/data4life/sdk/Data4LifeClient.java

```diff
@@ -139,7 +138,7 @@ public static Data4LifeClient init(byte[] accessToken,
                 platform,
                 networkConnectivityService,
                 NetworkingContract.Clients.INGESTION,
-                VERSION,
+                SDKConfig.version,
```

Comment:
@wmontwe here -> it does make use of it.

Reply:
👍